fedimint-cli/src/lib.rs

```diff
@@ -792,25 +793,29 @@ impl FedimintCli {
 
                 Ok(CliOutput::UntypedApiOutput { value: response })
             }
-            Command::Dev(DevCmd::WaitBlockCount { count: target }) => {
-                task::timeout(Duration::from_secs(60), async move {
+            Command::Dev(DevCmd::WaitBlockCount { count: target }) => retry(
+                "wait_block_count",
+                ConstantBackoff::default()
+                    .with_delay(Duration::from_millis(100))
+                    .with_max_times(usize::MAX),
+                || async {
                     let client = self.client_open(&cli).await?;
-                    loop {
-                        let wallet = client.get_first_module::<WalletClientModule>();
-                        let count = client
-                            .api()
-                            .with_module(wallet.id)
-                            .fetch_consensus_block_count()
-                            .await?;
-                        if count >= target {
-                            break Ok(CliOutput::WaitBlockCount { reached: count });
-                        }
-                        task::sleep(Duration::from_millis(100)).await;
+                    let wallet = client.get_first_module::<WalletClientModule>();
+                    let count = client
+                        .api()
+                        .with_module(wallet.id)
+                        .fetch_consensus_block_count()
+                        .await?;
+                    if count >= target {
+                        Ok(CliOutput::WaitBlockCount { reached: count })
+                    } else {
+                        Err(format_err!("target not reached"))
```

Comment:
Thought: should we `info!(...)` something here (e.g. target, current, "Block count not reached").

Reply:
done